src/common/libflux/message.h

```diff
@@ -191,7 +191,7 @@ int flux_msg_set_flags (flux_msg_t *msg, uint8_t flags);
 /* Get/set string payload.
  * flux_msg_set_string() accepts a NULL 's' (no payload).
  * flux_msg_get_string() will set 's' to NULL if there is no payload
- * N.B. the raw paylaod includes C string \0 terminator.
+ * N.B. the raw payload includes C string \0 terminator.
```

Comment:
commit message typo? I see only one comment misspelling, not two :-)

Reply:
Huh, weird.  I could have sworn there were two.  Oh, found the same misspelled word in event.h.  I'll tack that on and fix the commit message.